modules/andes-core/broker/src/main/java/org/wso2/andes/server/information/management/QueueManagementInformationMBean.java

```diff
@@ -61,10 +71,11 @@
 import java.nio.charset.CharacterCodingException;
 import java.nio.charset.Charset;
 import java.util.ArrayList;
-import java.util.Arrays;
 import java.util.Iterator;
 import java.util.List;
 import java.util.Map;
+import java.util.Collections;
+
 
 public class QueueManagementInformationMBean extends AMQManagedObject implements QueueManagementInformation {
```

Comment:
Addressed by the PR https://github.com/wso2/andes/pull/403/files. Corrected in identified classes. Will be looking into others as well.